_includes/index.md

```diff
@@ -6,7 +6,7 @@ A campaign for Mozillians around the world to have impact in areas key to Mozill
 ## Focus Initiatives
 
 {: #area-deveng .area}
-[![image](/asserts/img/development.png)
+[![image]({{ "/asserts/img/development.png" | prepend: site.baseurl }})
```

Comment:
Yes, so probably was the time it took to show changes on the server. But you could see the previous change was made by me too, because I prefer that version. So I will make another change reflecting the old behavior.